src/index.ts

```diff
@@ -154,4 +140,21 @@ module.exports = class ServerlessPlugin {
       }
     });
   }
+
+  private getHandlers(nodeModuleType: "es6" | "node" | "typescript" | undefined) {
+    const defaultRuntime = this.serverless.service.provider.runtime;
+    let defaultNodeRuntime: RuntimeType.NODE | RuntimeType.NODE_ES6 | RuntimeType.NODE_TS | undefined;
+    switch (nodeModuleType) {
+      case "es6":
```

Comment:
The es6 and TS types were added to help us use different handler templates  when using webpack/typescript plugins. They all map to a regular node runtime. Not sure if they are necessary anymore